app/models/reports/answer_tagging_report.rb

```
@@ -0,0 +1,233 @@
+
+      def run
+        summary = {}
+        @per_deployment.each_value do |deployment_data|
+          deployment_data[:user_stats].each do |stat|
+            key = stat[:user_id]
+            if summary.key?(key)
+              entry = summary[key]
+              entry[:cnt_tagged]     += stat[:cnt_tagged]
+              entry[:cnt_not_tagged] += stat[:cnt_not_tagged]
+              entry[:cnt_taggable]   += stat[:cnt_taggable]
+              entry[:percentage] = entry[:cnt_taggable].zero? ? '-' : format('%.1f', entry[:cnt_tagged].to_f / entry[:cnt_taggable] * 100)
```

RuboCop (1.86.2)

[convention] Line is too long. [138/120]

(Layout/LineLength)